opensearch/src/cert.rs

```diff
@@ -233,7 +233,10 @@ impl Certificate {
                 END_CERTIFICATE if begin => {
                     begin = false;
                     cert.push(line);
-                    certs.push(reqwest::Certificate::from_pem(cert.join("\n").as_bytes())?);
+                    certs.push(
+                        reqwest::Certificate::from_pem(cert.join("\n").as_bytes())
+                            .map_err(CertificateError::MalformedCertificate)?,
```

Comment:
Super nit: 

```suggestion
                            .map_err(CertificateError::MalformedCertificate)?
```

Reply:
This is imposed by `cargo fmt` due to how it decides to break the line. Only way around it is to restructure the line entirely.